gen/comp/door_slider

```diff
@@ -0,0 +1,4 @@
+ui_name,Door slider
```

Comment:
door_slider or linear_actuator?

Reply:
I'm not trying to make this generic -- it looks directly into a parent door component.